ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/server/execution/PullQueryMetricsTest.java

```diff
@@ -99,9 +98,9 @@ public void shouldRecordNumberOfLocalRequests() {
 
     // Then:
     assertThat(value, equalTo(1.0));
-    assertThat(rate, closeTo(0.03, 0.001));
+    assertThat(rate, greaterThan(0.0));
```

Comment:
These tests were failing because the measured rates were slightly outside of the tolerances of the prior test. All we really care about is that the rate metric got registered and recorded, so `>0` is sufficient. I think the measurement changed just due to the difference in the amount of time that passed between the creation of the static registry before and this test's registry instance now.